[{"timestamp":"2021-01-06T01:03:27.020Z","id":"49b775e3-e2ab-4538-9f4b-70f2fd8c8dbf","type":"task","name":"acquireFakeGeolocation","result":"error","content":{"took":4,"invokedBy":"taskExecutionStarted","message":"Error: Iteration cycle is no longer aligned to data (dataSource=single-track,instant=190574)\n    at new MisalignedDataError (file:///data/user/0/es.uji.geotec.symptomssimulationapp/files/app/bundle.js:295:23)\n    at FakeDataProvider.<anonymous> (file:///data/user/0/es.uji.geotec.symptomssimulationapp/files/app/bundle.js:418:39)\n    at step (file:///data/user/0/es.uji.geotec.symptomssimulationapp/files/app/vendor.js:81062:23)\n    at Object.next (file:///data/user/0/es.uji.geotec.symptomssimulationapp/files/app/vendor.js:81043:53)\n    at fulfilled (file:///data/user/0/es.uji.geotec.symptomssimulationapp/files/app/vendor.js:81033:58)"}}]